src/NHibernate/Hql/Ast/ANTLR/QueryTranslatorImpl.cs

```diff
@@ -652,7 +652,7 @@ public IStatement Translate()
 				try
 				{
 					// Transform the tree.
```

Comment:
This comment is obsolete too. (And was not really useful, the code was already saying that.)

```suggestion
```

Reply:
Ok. But it's some old comment (maybe "ported" from hibernate). It wasn't added in #2964